pkg/addons/addons.go

```diff
@@ -201,13 +204,20 @@ https://github.com/kubernetes/minikube/issues/7332`, out.V{"driver_name": cc.Dri
 		}
 	}
 
-	cmd, err := machine.CommandRunner(host)
+	runner, err := machine.CommandRunner(host)
 	if err != nil {
 		return errors.Wrap(err, "command runner")
 	}
 
+	if name == "auto-pause" && !enable { // needs to be disabled before deleting the service file in the internal disable
+		cmd := exec.Command("sudo", "systemctl", "disable", "--now", "auto-pause")
```

Comment:
I'd replace this with `sysinit.New(runner).Disable("auto-pause")` so that we can take advantage of existing libraries

Reply:
the problem is our sysinit does not have Disable Now (but I could add it as part of this)

Reply:
done